tests/testdata/Dockerfile-postgis

```diff
@@ -0,0 +1,9 @@
+FROM kartoza/postgis:11.0-2.5
+
+ADD auth_system/certs_keys/postgres.crt /etc/ssl/certs/postgres_cert.crt
+ADD auth_system/certs_keys/postgres.key /etc/ssl/private/postgres_key.key
+ADD auth_system/certs_keys/issuer_ca_cert.pem /etc/ssl/certs/issuer_ca_cert.pem
+
+RUN chmod 400 /etc/ssl/private/postgres_key.key
+
+ADD temp/setup-ssl.sh /
```

Comment:
Are you sure this setup-ssl.sh is actually run inside the container?

Reply:
Pretty sure:

* That's how the upstream package works
* It sets up ssl properly when upping this docker compose locally here
* Looking at the "/root/.postgresql/root.crt" error on travis it much more looks like a client- (test-)side problem than a server config problem